### PR TITLE
#2246 Multiple bond editing changes bond types to all selected bonds

### PR DESCRIPTION
**Pull request by AnastasiiaPlyako**
closes https://github.com/epam/ketcher/issues/2246

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/data/schema/struct-schema.js [hunk 1]
```diff
@@ -155,6 +155,7 @@ export const bond = {
     type: {
       title: 'Type',
       enum: [
+        '',
```
Comment (Konstantin1996) [inline packages/ketcher-react/src/script/ui/data/schema/struct-schema.js:158]: Looks strange

**Review by Konstantin1996 — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/data/schema/struct-schema.js (hunk 1 above)
Comment (KonstantinEpam23) [reply packages/ketcher-react/src/script/ui/data/schema/struct-schema.js:158]: @AnastasiiaPlyako can you please comment on this

Comment (AnastasiiaPlyako) [reply packages/ketcher-react/src/script/ui/data/schema/struct-schema.js:158]: Discussed with Konstantin. This field allows you to have an empty line when selected bonds have different types. The same logic is realized for atoms. https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/data/schema/struct-schema.js#L62

File: packages/ketcher-react/src/script/ui/data/convert/structconv.js [hunk 2]
```diff
@@ -232,7 +232,7 @@ function fromBondType(type, stereo) {
     )
       return caption
   }
-  throw Error('No such bond caption')
+  return ''
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/data/convert/structconv.js:235]: Could you please clarify, why this is needed? :)

**Review by KonstantinEpam23 — APPROVED** [review]

**Pull request merged**